test/integration/targets/systemd/tasks/main.yml

```diff
@@ -112,6 +112,32 @@
     enabled: true
   register: systemd_enable_ssh_2
 
+- name: get tftp package to test indirect service
+  package:
+    name: '{{ tftp_package }}'
+    state: present
+  when: (ansible_distribution == "CentOS" or ansible_distribution == "RedHat") and ansible_distribution_version|int < 9
```

Comment:
What's the ansible_distribution_version restriction is for? Do the tests fail on RHEL 9.0b?

Reply:
I think this may have been from the tftp package not being in the repo.. I will remove this next push and see exactly why.

Reply:
I ended up going the route to just creating own indirect dummy service in integration tests, hopefully this takes care of all OS's